pkg/controller/admissionchecks/provisioning/controller_test.go

```diff
@@ -661,19 +628,7 @@ func TestReconcile(t *testing.T) {
 			workload: baseWorkload.DeepCopy(),
 			checks:   []kueue.AdmissionCheck{*baseCheck.DeepCopy()},
 			flavors:  []kueue.ResourceFlavor{*baseFlavor1.DeepCopy(), *baseFlavor2.DeepCopy()},
-			configs: []kueue.ProvisioningRequestConfig{
-				{ObjectMeta: metav1.ObjectMeta{
-					Name: "config1",
-				},
-					Spec: kueue.ProvisioningRequestConfigSpec{
-						ProvisioningClassName: "class1",
-						Parameters: map[string]kueue.Parameter{
-							"p1": "v1",
-						},
-						ManagedResources: []corev1.ResourceName{corev1.ResourceMemory},
-					},
-				},
-			},
+			configs:  []kueue.ProvisioningRequestConfig{*baseConfig.WithManagedResource(corev1.ResourceMemory).DeepCopy()},
```

Comment:
```suggestion
			configs:  []kueue.ProvisioningRequestConfig{baseConfig.Clone().WithManagedResource(corev1.ResourceMemory).Obj()},
```
Please follow this pattern with the rest of changes